qiita_db/sql_connection.py

```diff
@@ -646,9 +646,12 @@ def _raise_execution_error(self, sql, sql_args, error):
         """
         self.rollback()
 
-        raise ValueError(
-            "Error running SQL: %s. MSG: %s\n" % (
-                errorcodes.lookup(error.pgcode), error.message))
+        try:
```

Comment:
This block seems rather confusing, you `try` to find the error code, and check if that raises a `KeyError`, if it doesn't you show the error and the message, and if it does you just show the message. Are pgcodes not guaranteed to be in the errorcodes?

Reply:
I found one case in which pgcodes was not in the error codes. It was mainly due to a bad usage of psycopg2 and the multiprocessing module. That is now fixed, but without this try/except I was unable to see the error message. Thus, in general they will be found, but just in case that in the future we find another issue, this will allow us to debug.

Reply:
Wow, that's very strange!